blackbox/docs/sql/administration/privileges.txt

```diff
@@ -133,8 +133,8 @@ information regarding the existing privileges. E.g.::
     +---------+---------+---------+-------+-------+------+
     | class   | grantee | grantor | ident | state | type |
     +---------+---------+---------+-------+-------+------+
-    | CLUSTER | will    | crate   |  NULL | GRANT | DQL  |
     | CLUSTER | will    | crate   |  NULL | GRANT | DML  |
+    | CLUSTER | will    | crate   |  NULL | GRANT | DQL  |
```

Comment:
yes, we're adding an order by as part of the read-only story (will get in the parent branch)